Java math cbrt entry

cryoung1998 opened the pull request:
<!--
👋 Hi, thanks for sending a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description

<!-- Please write a summary of the change, such as which topic(s) and file(s) that you have edited or created. Please also include relevant motivation and context: -->
Created a Math.cbrt() method documentation file for the Codecademy Community Docs. Added a single file cbrt.md with pertinent information on the Math.cbrt() Java method.

### Type of Change

<!--- Please delete or cross off the bullet point(s) that are irrelevant to this PR: -->

- Adding a new entry

### Checklist

<!-- Please check ALL the boxes: -->

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Codecademy/docs?pullRequest=1202) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=1202) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Codecademy/docs?pullRequest=1202) it.</sub>

cryoung1998 commented:
#1202 Linking to assigned Issue number

SSwiniarski commented on content/java/concepts/math-methods/terms/cbrt/cbrt.md:
```diff
@@ -0,0 +1,52 @@
+  - 'path/computer-science'
+---
+
+The **`Math.cbrt()`** method returns the cubed root of a double value.
```
```suggestion
The **`Math.cbrt()`** method returns the cubed root of a `double` value.
```

cryoung1998 commented:
Good catch! I commited the revision and it should reflect the update in the code now

SSwiniarski reviewed (approved):
@cryoung1998 Looks good for review two.

yangc95 commented on content/java/concepts/math-methods/terms/cbrt/cbrt.md:
```diff
@@ -0,0 +1,52 @@
+---
+Title: '.cbrt()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Returns the cubed root of a double-type value.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
+  - 'Data Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Arithmetic'
+  - 'Documentation'
+  - 'Functions'
+  - 'Numbers'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
```
Let's remove these comments within the metadata~ They only exist in the templates as clarification

yangc95 commented on content/java/concepts/math-methods/terms/cbrt/cbrt.md:
````diff
@@ -0,0 +1,52 @@
+Math.cbrt(double num)
+```
+
+The `Math.cbrt()` method will return the cube root of `num`. The value of `num` can be any real number. Edge cases for the `.cbrt()` function include:
````
```suggestion
The `.cbrt()` method will return the cube root of `num`. The value of `num` can be any real number. Edge cases for this method include:

```

yangc95 commented on content/java/concepts/math-methods/terms/cbrt/cbrt.md:
````diff
@@ -0,0 +1,52 @@
+```
+
+The `Math.cbrt()` method will return the cube root of `num`. The value of `num` can be any real number. Edge cases for the `.cbrt()` function include:
+- If the argument is `NaN`, then `.cbrt()` will return `NaN`
````
```suggestion
- If the argument is `NaN`, the return value will be `NaN`.
```

yangc95 commented on content/java/concepts/math-methods/terms/cbrt/cbrt.md:
```diff
@@ -0,0 +1,52 @@
+
+The `Math.cbrt()` method will return the cube root of `num`. The value of `num` can be any real number. Edge cases for the `.cbrt()` function include:
+- If the argument is `NaN`, then `.cbrt()` will return `NaN`
+- If the argument is ±infinity, then `.cbrt()` will return ±infinity
```
```suggestion
- If the argument is ±infinity, the return value will be ±infinity.
```

yangc95 commented on content/java/concepts/math-methods/terms/cbrt/cbrt.md:
```diff
@@ -0,0 +1,52 @@
+
+## Example
+
+Here is an example of the `Math.cbrt()` method:
```
```suggestion
Here is an example using the `Math.cbrt()` method:
```

yangc95 commented on content/java/concepts/math-methods/terms/cbrt/cbrt.md:
````diff
@@ -0,0 +1,52 @@
+```java
+public class Main {
+    public static void main(String[] args) {
+
````
```suggestion
```

yangc95 commented on content/java/concepts/math-methods/terms/cbrt/cbrt.md:
```diff
@@ -0,0 +1,52 @@
+
+        double b = -8.0;
+        System.out.println( "Math.cbrt("+b") returns: " + Math.cbrt(b) );
+
```
```suggestion
```

yangc95 commented on content/java/concepts/math-methods/terms/cbrt/cbrt.md:
```diff
@@ -0,0 +1,52 @@
+    public static void main(String[] args) {
+
+        double a = 27.0;
+        System.out.println( "Math.cbrt("+a") returns: " + Math.cbrt(a) );
```
```suggestion
        System.out.println("Math.cbrt("+a") returns: " + Math.cbrt(a));
```

yangc95 commented on content/java/concepts/math-methods/terms/cbrt/cbrt.md:
```diff
@@ -0,0 +1,52 @@
+        System.out.println( "Math.cbrt("+a") returns: " + Math.cbrt(a) );
+
+        double b = -8.0;
+        System.out.println( "Math.cbrt("+b") returns: " + Math.cbrt(b) );
```
```suggestion
        System.out.println("Math.cbrt("+b") returns: " + Math.cbrt(b));
```

yangc95 reviewed (commented):
Well-written PR @cryoung1998 ! I conducted a second review.

Please tag me if you have any questions and when you've finished resolving all the comments/suggestions.

cryoung1998 commented:
Thanks for the thorough reveiw @yangc95 ! I implemented all of the changes and removed the comments from the code.

yangc95 reviewed (approved):
LGTM. I had to make a change here: https://github.com/Codecademy/docs/pull/1202/commits/383bd84d562eb72186909f4fd3088f5445989066

We forgot a `+` in the output (was giving an error).

Dusch4593 commented:
Congrats and thank you for contributing to Codecademy Docs! 🎊 🎉 

The entry should now be live: https://www.codecademy.com/resources/docs/java/math-methods/cbrt

Currently, your profile is listed anonymously on the entry page. To make it visible, you'd need to add your GitHub username to your Codecademy profile, log out of Codecademy, and log back in through your GitHub account. Let us know if you'd like assistance with doing this. 😄

<img width="995" alt="Screen Shot 2022-11-08 at 11 29 39 AM" src="https://user-images.githubusercontent.com/23144845/200621362-aaaa170f-2747-4d9c-a603-49ab4b1c3ad4.png">

Dusch4593 commented:
@cryoung1998 I was also wondering if you'd be open to creating a new PR to make some edits to this entry? My suggestions would include the following:

- On line # 3, let's change "cubed root" to "cube root."
- On line # 17, let's also change "cubed root" to "cube root."
- In the \```java code block under the "Example" section, let's change the indents from 4 spaces to 2 spaces.

cryoung1998 commented:
@Dusch4593 I opened the new PR #1229 where I implemented the changes you suggested! I also added my github account to my codeCademy account and that should be linked now. Also, great suggestions, I would have never caught "cubed root", turns out I've been saying it wrong for a long time.

Dusch4593 commented:
> @Dusch4593 I opened the new PR #1229 where I implemented the changes you suggested! I also added my github account to my codeCademy account and that should be linked now. Also, great suggestions, I would have never caught "cubed root", turns out I've been saying it wrong for a long time.

Thank you, @cryoung1998! I recently merged #1229 😄

To follow-up on the GitHub account, I'm still not seeing your profile displaying on the [`.cbrt()` entry](https://www.codecademy.com/resources/docs/java/math-methods/cbrt) (can you?).

I was wondering if you also logged out of Codecademy and re-logged in through your GitHub account?

cryoung1998 commented:
@Dusch4593 Should be all set now, not sure what the hold up was but I opened my account, verified the correct user, and then hit save changes and my user appeared on the entry 🤷 Thanks for all of your help!

Dusch4593 commented:
Yup, I'm seeing you now, to! Thanks, @cryoung1998! 🚀 

<img width="907" alt="Screen Shot 2022-11-09 at 7 11 46 PM" src="https://user-images.githubusercontent.com/23144845/200968935-413788e4-e1ac-4115-adb4-bea2633b98a8.png">
